docs: update quickstart-cli documentation and add images for tsci push (#34)
tscircuit/docs: docs/command-line/tsci-push.md

---
title: tsci push
sidebar_position: 2
---

> `tsci push` pushes your snippet code to Registry API.

Reviewer: ```suggestion
tscircuit code, like React code, is distributed as a "package". `tsci push` pushes your package to [tscircuit registry](https://tscircuit.com/trending) to enable sharing with your team.

After publishing, you can use the [tsci add](https://docs.tscircuit.com/command-line/tsci-add) command to install your package as part of a larger circuit.
```

---
title: tsci push

> sidebar_position: 2

Reviewer: ```suggestion
sidebar_position: 2
description: |
  tsci push uploads your package to the tscircuit registry for module re-use
```

---

> tscircuit code, like React code, is distributed as a "package". `tsci push` pushes your package to [tscircuit registry](https://tscircuit.com/trending) to enable sharing with your team.

Reviewer: ```suggestion
tscircuit code, like React code, is distributed as a "package". `tsci push` uploads your package to [tscircuit registry](https://tscircuit.com/trending) to enable sharing with your team.
```

After publishing, you can use the [tsci add](https://docs.tscircuit.com/command-line/tsci-add) command to install your package as part of a larger circuit.

> After you run `tsci push` you can see a your snippet file on your tscircuit account.

Reviewer: ```suggestion
After running `tsci push` you can see your package on your tscircuit registry page. Packages default to private visibility, but you can change this from your registry page to enable sharing your package with the broader ecosystem.
```

---
title: tsci push
sidebar_position: 2
description: |
  tsci push uploads your package to the tscircuit registry for module re-use
---

> tscircuit code, like React code, is distributed as a "package". `tsci push` uploads your package to [tscircuit registry](https://tscircuit.com/trending) to enable sharing with your team.

Reviewer: ```suggestion
tscircuit code, like React code, is distributed as a "package". `tsci push` uploads your package to the [tscircuit registry](https://tscircuit.com/trending) to enable sharing with your team.
```
